packages/react-instantsearch-dom-geo/src/GeoSearch.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+// @TODO: Update this import when the package is correctly split:
+// import { createClassNames } from 'react-instantsearch-dom';
+import createClassNames from '../../react-instantsearch/src/components/createClassNames';
```

Comment:
With an absolute import the file will be the original one (non transpiled). Webpack will complain because we don't the `node_modules` files are not transpiled. In the next version this issue is fixed with a `watch` command that build the packages. For now, we can let the import like that.